### ISSUE TITLE
zos showdefault command

### ISSUE THREAD

**Issue by Dina-Abd-Elrahman** [threefoldtecharchive/zosctl#73]
- we need to add command ``` zos showdefault``` to show default node we are using now.

**Comment by xmonader** [comment]
the node name and the connection string too instead of doing greps around the zos.toml

**Comment by xmonader** [comment]
Add two commands (showactive to show instance name and showactiveconfig to show active configurations zos configured against) 
fixed here 74e150d15cc7b01cf80230697a988f5e440030a1

**Comment by Dina-Abd-Elrahman** [comment]
# verification steps:

- ./zos showactive

```test-3```

- ./zos showactiveconfig

```
{"address": "127.0.0.1", "port": "9986", "isvbox": "true", "lastsshport": "19002"}

```
